src/components/Pagination/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import cn from "classnames";
+import "./index.scss";
+
+const Pagination = ({
+  prevPageIndex,
+  nextPageIndex,
+  updatePageIndex,
+  pageIndex,
+  numRules,
+}) => {
+  const endPage = pageIndex + (5 - (pageIndex % 5 === 0 ? 5 : pageIndex % 5));
+  const startPage = endPage - 4;
+  return (
+    <div className="pagination pagination-container">
+      <a
+        className="btn btn-secondary btn-small pagination-prev"
+        onClick={prevPageIndex}
+        onKeyDown={() => {}}
```

Comment:
if this won't be used it can be removed. unless it yells at you for removing it

Reply:
but yes it does yell at me haha